test/mitmproxy/console/test_flowlist.py

```diff
@@ -0,0 +1,21 @@
+import mitmproxy.tools.console.flowlist as flowlist
+from mitmproxy.tools import console
+from mitmproxy import proxy
+from mitmproxy import options
+from .. import tservers
+from unittest import mock
+
+
+class TestFlowlist(tservers.MasterTest):
```

Comment:
Do we even need the `tservers.MasterTest` parent? We are building our own master anyway...
If not, please remove it.